tools/DB_date_zeros_removal.php

```diff
@@ -14,35 +14,28 @@
 set_include_path(get_include_path().":".__DIR__."/../project/libraries:".":".__DIR__."/../../php/libraries:");
 require_once __DIR__ . "/../vendor/autoload.php";
 //require_once "NDB_Config.class.inc";
-
```

Comment:
even though the changes in this file aren't really doing anything, you might want to `git checkout --` this file for the sake of best practices.
actual changes above are all good though